run_gemini.py

```diff
@@ -0,0 +1,64 @@
+import google.generativeai as genai
+import os
+import subprocess
+import logging
+from dotenv import load_dotenv
+
+
+def get_review(api_key, commit_branch, main_branch, logger):
+    subprocess.run(
+        f"git fetch origin {main_branch} && git fetch origin {commit_branch} && git checkout {commit_branch}",
+        shell=True,
+        check=True,
+    )
+    git_diff = subprocess.run(
+        f"git diff origin/{main_branch}..origin/{commit_branch}",
+        shell=True,
+        stdout=subprocess.PIPE,
+        stderr=subprocess.PIPE,
+        check=True,
+    )
+    if git_diff.stderr != b"":
+        logger.error(f"stderr:{git_diff.stderr}")
```

Comment:
The script assumes a consistent `git diff main..HEAD` format, but it might not be always the case, and the script could break if the format changes.